src/sentry/relay/config.py

```diff
@@ -3,15 +3,296 @@
 import re
 import six
 import uuid
-import json
+import sentry.utils as utils
 
 from datetime import datetime
 from pytz import utc
 
-from sentry.models import ProjectKey, OrganizationOption
+from sentry import options
+from sentry.coreapi import APIError
+from sentry.grouping.api import get_grouping_config_dict_for_project
+from sentry.message_filters import get_all_filters
+
+from sentry.models.organization import Organization
+from sentry.models.organizationoption import OrganizationOption
+from sentry.models.project import Project
+from sentry.models.projectoption import ProjectOption
+from sentry.utils.data_filters import FilterTypes, FilterStatKeys
+from sentry.utils.http import get_origins
+from sentry.utils.outcomes import track_outcome, Outcome
+from sentry.models.projectkey import ProjectKey
 from sentry.utils.sdk import configure_scope
 
 
+def get_project_options(project):
+    """
+    (Deprecated) Returns a minimal relay config dictionary
```

Comment:
This would only be used in the (also deprecated) `get_project_options` query. Since this is effectively dead code, let's inline it with `get_relay_config`, which should be called `get_project_config`.

Reply:
done